### DISCUSSION TITLE
Import size

### DISCUSSION THREAD

**Discussion by hetari** [motdotla/dotenv#823] [Q&A] [answered]
I'm using [import cost](https://marketplace.visualstudio.com/items?itemName=wix.vscode-import-cost) extension in vscode. 

I am trying to import `dotenv` and have only used the config method. However, when I wanted to import it only in the second line, expecting a smaller size, I ended up with almost the same size. I am confused about why this happened and whether I have misunderstood the concept of import size. I know that importing only a part of the module should result in a smaller size. Can you please help me understand if I got this wrong?

![image](https://github.com/motdotla/dotenv/assets/108173819/9e47cb4d-0b3d-4b9c-8523-f02ccdf9c140)

**Comment by motdotla** [comment, marked as answer]
dotenv was designed before there were imports and it was always just a single file. the dotenv/config actually loads the config file with loads a couple additionally methods. that's why you are seeing it actually larger. it's so small a difference in size though it's not been a priority for us to adjust that.
